packages/igx-templates/igx-ts/dock-manager/default/files/src/app/__path__/__filePrefix__.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import { IgcDockManagerLayout,
+	IgcDockManagerPaneType,
+	IgcSplitPaneOrientation
+  } from "igniteui-dockmanager";
```

Comment:
All though double quotes should be single, if you run lint in a created project:
```
ERROR: ./src/app/dock-manager/dock-manager.component.ts:5:10 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:26:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:27:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:31:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:32:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:54:57 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:55:60 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:59:57 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:60:60 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:69:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:70:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:85:48 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:86:45 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:90:48 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:91:45 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:97:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:98:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:114:36 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:115:33 - " should be 'ERROR: ./src/app/dock-manager/dock-manager.component.ts:5:10 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:26:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:27:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:31:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:32:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:54:57 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:55:60 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:59:57 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:60:60 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:69:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:70:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:85:48 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:86:45 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:90:48 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:91:45 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:97:40 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:98:37 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:114:36 - " should be '
ERROR: ./src/app/dock-manager/dock-manager.component.ts:115:33 - " should be '
```